specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -678,7 +684,7 @@
         },
         "type": {
           "type": "string",
-          "description": "Determines whether the IP is exposed to the public internet.",
+          "description": "The type of IP address determining whether the IP is exposed to the public internet.",
           "enum": [
             "Public"
           ],
```

Comment:
On Line 709 - I don't think you can actually specify a UDP port today. We should remove this bit if we generate docs before it is supported.

Reply:
Do we know when UDP will be supported, we can leave the documentation for it if it is very soon.